### ISSUE TITLE
All Souls' Masses are broken with 1960 rubrics

### ISSUE THREAD

**Issue by GoogleCodeExporter** [DivinumOfficium/divinum-officium#77]
```
http://divinumofficium.com/cgi-bin/missa/missa.pl?date=11-2-2011&version=Rubrics
%201960

r187 added separate 1960 versions (11-02r.txt) for the office, but not for 
Mass. This led to 2nd Nov being identified as a feria under the 1960 rubrics. 
r206 was my half-boiled attempt to fix it: it created *some* of the 
corresponding files for Mass, but neglected to account for the three Masses of 
the day (11-02m?.txt). The result was a strange mix of the Masses, regardless 
of which was selected.

Rather than duplicating those files too, implementation of data-file 
conditionals as specified in issue #76 would allow all the logic for the office 
to be expressed in one file per language, removing the need for separate 1960 
files at all.
```

Original issue reported on code.google.com by `igregord` on 16 Nov 2011 at 1:59
- Blocked on: #76
- Merged into: #64
